apps/mymedicare_cb/models.py

```diff
@@ -27,14 +27,26 @@ def get_and_update_user(user_info):
     """
     subject = user_info['sub']
     hicn = user_info['hicn']
+    mbi = user_info['mbi']
+
+    # If mbi is empty set to None
+    if mbi == "":
+        mbi = None
+
+    # Create hashed values.
     hicn_hash = hash_hicn(hicn)
+    mbi_hash = hash_mbi(mbi)
 
+    # TODO: Add return for ID type used for FHIR_ID match.
     # raises exceptions.NotFound:
     fhir_id, backend_data = match_hicn_hash(hicn_hash)
 
+    # HICN was used to match the user.
+    user_id_type = 'H'
+
```

Comment:
Currently only a HICN is used for the matching. For the next phase, this will be set to the method that was used for the patient resource lookup (H or M).